dlc-manager/src/manager.rs

```diff
@@ -827,6 +810,57 @@ where
         Ok(())
     }
 
+    /// Check for pending cooperative close transactions
+    fn check_pending_close_transactions(&self) -> Result<(), Error> {
```

Comment:
This logic should be implemented within `check_confirmed_contracts`, unless there is a reason to have it separate? Right now that means having to go two times through the confirmed contracts.